### PR TITLE
Added Italian translation of leaf/overview.md and basics/client.md

### PR DESCRIPTION
**Pull request by fpseverino**
- Added Italian translation of leaf/overview.md and basics/client.md
- Added Controllers to the navigation components in Italian
- Fixed typo in English version of leaf/overview.md (Leaf was not capitalized)

### CODE REVIEW

File: docs/leaf/overview.it.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+#endfor
+```
+
+## Utilizzo
+
+Ecco alcuni esempi di utilizzo comune di Leaf.
+
+### Condizioni
+
+Leaf è capace di valutare una serie di condizioni usando il suo tag `#if`. Per esempio, se gli fornisci una variabile controllerà che la variabile esiste nel suo contesto:
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:81]:
```suggestion
Leaf è capace di valutare una serie di condizioni usando il suo tag `#if`. Per esempio, se gli fornisci una variabile controllerà che la variabile esista nel suo contesto:
```

File: docs/leaf/overview.it.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+```leaf
+#if(title):
+    The title is #(title)
+#else:
+    No title was provided.
+#endif
+```
+
+Puoi anche scrivere confornti, per esempio:
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:91]:
```suggestion
Puoi anche utilizzare confronti, per esempio:
```

File: docs/leaf/overview.it.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+#elseif(title == "Welcome back!"):
+    Hello old user
+#else:
+    Unexpected page!
+#endif
+```
+
+### Cicli
+
+Se fornisci un array di oggetti, Leaf può iterare su essi e permetterti di manipolare ciascun oggetto individualmente usando il suo tag `#for`.
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:125]:
```suggestion
Se fornisci un array di oggetti, Leaf può iterare su di essi e permetterti di manipolare ciascun oggetto individualmente usando il suo tag `#for`.
```

File: docs/leaf/overview.it.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+
+Per esempio, possiamo aggiornaree il nostro codice Swift per fornire una lista di pianeti:
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:127]:
```suggestion
Per esempio, possiamo aggiornare il nostro codice Swift per fornire una lista di pianeti:
```

File: docs/leaf/overview.it.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+
+```swift
+struct SolarSystem: Codable {
+    let planets = ["Venus", "Earth", "Mars"]
+}
+
+return req.view.render("solarSystem", SolarSystem())
+```
+
+Possiamo iterare su essi in Leaf così:
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:137]:
```suggestion
In Leaf, possiamo iterare su di essi così:
```

File: docs/leaf/overview.it.md [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+
+```swift
+render(..., ["now": Date()])
+```
+
+```leaf
+The time is #date(now)
+```
+
+Puoi passare una stringa di formato di data personalizzato come secondo argomento. Guarda il [`DateFormatter`](https://developer.apple.com/documentation/foundation/dateformatter) di Swift per più informazioni.
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:261]:
```suggestion
Puoi passare una stringa in un formato di data personalizzato come secondo argomento. Guarda il [`DateFormatter`](https://developer.apple.com/documentation/foundation/dateformatter) di Swift per più informazioni.
```

File: docs/leaf/overview.it.md [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+
+```leaf
+The date is #date(now, "yyyy-MM-dd")
+```
+
+#### `#unsafeHTML`
+
+Il tag `#unsafeHTML` agisce come un tag di variabile - e.g. `#(variable)`. Però non evade nessun HTML che `variable` potrebbe contenere:
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:269]:
```suggestion
Il tag `#unsafeHTML` agisce come un tag di variabile - p.es. `#(variable)`. Però non evade nessun HTML che `variable` potrebbe contenere:
```

File: docs/leaf/overview.it.md [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+
+```leaf
+The time is #unsafeHTML(styledTitle)
+```
+
+!!! note
+    Dovresti fare attenzione quando usi questo tag per assicurarti che la variabile che gli fornisci non espone i tuoi utenti a un attacco XSS.
```
Comment (ptoffy) [inline docs/leaf/overview.it.md:276]:
```suggestion
    Dovresti fare attenzione quando usi questo tag per assicurarti che la variabile che gli fornisci non esponga i tuoi utenti a un attacco XSS.
```

**Review by ptoffy — CHANGES_REQUESTED** [review]
Just a few nits but feel free to merge afterwards!

**Review by fpseverino — COMMENTED** [review]
All requested chances done

**Review by ptoffy — APPROVED** [review]

**Pull request merged**